### PR TITLE
BUILD: Add news file

### PR DESCRIPTION
**Pull request by manjugv**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  layer
+- Support for basic collectives types including barrier, alltoall, alltoallv,
+  broadcast, allgather, allgatherv, allreduce was added in the UCP TL
+- Added support using NCCL as a transport layer
+- Support for collectives types including alltoall, alltoallv, allgather,
+  allgatherv, allreduce, and broadcast
+
+#### Tests
+
+- Added support for unit (gtest) infrastructure
```
Comment (Sergei-Lebedev) [inline NEWS:43]: Added support for unit testing (gtest) infrastructure

File: NEWS [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## 1.0.0 (TBD)
+
+### Features
+
+#### API
+- UCC API to support library, contexts, teams, collective operations, execution
+  engine, memory types, and triggered operations
+
+#### Core
+- Added implementation for UCC abstractions - library, contexts, team,
```
Comment (Sergei-Lebedev) [inline NEWS:16]: library, context, team

File: NEWS [hunk 3]
```diff
@@ -0,0 +1,44 @@
+/**
+ * @copyright Copyright (C) Mellanox Technologies Ltd. 2021.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+## 1.0.0 (TBD)
```
Comment (Sergei-Lebedev) [inline NEWS:7]: 0.5.0 ?

File: NEWS [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  operations
+- Added support for memory types - CUDA, and CPU
+- Added support for configuring UCC library and contexts
+
+
+#### CL 
+
+- Added support for collectives, while the source and destination is either in
+  CPU or device (GPU) 
+- Added support for UCC_THREAD_MULTIPLE_MODE
```
Comment (Sergei-Lebedev) [inline NEWS:27]: UCC_THREAD_MULTIPLE mode

**Review by Sergei-Lebedev — APPROVED** [review]

**Comment by Sergei-Lebedev** [comment]
@manjugv should we also specify some recommended UCX version or commit to use?

**Review by vspetrov — APPROVED** [review]

**Comment by manjugv** [comment]
> @manjugv should we also specify some recommended UCX version or commit to use?

I think this should belong to FAQ section. I started a wiki page for FAQ. Currently, it only has questions section.

**Pull request merged**